hotham/src/contexts/xr_context/mod.rs

```diff
@@ -464,6 +464,10 @@ fn enable_xr_extensions(required_extensions: &mut xr::ExtensionSet) {
 #[cfg(not(target_os = "android"))]
 fn enable_xr_extensions(required_extensions: &mut xr::ExtensionSet) {
     required_extensions.khr_vulkan_enable = true;
+
+    if cfg!(feature = "editor") {
```

Comment:
I there any reason to use `if cfg!(..` instead of `#[cfg(..`?

Reply:
Can't do attributes on statements, apparently. 🤷